packages/astro/src/@types/astro.ts

```diff
@@ -1481,6 +1484,7 @@ export interface AstroUserConfig {
 
 			/**
 			 * @docs
+		   * @kind h4
 			 * @name experimental.i18n.fallback
 			 * @type {Record<string, string>}
 			 * @version 3.5.0
```

Comment:
Now that this is an `<h4>`, the `#### Example` in the `@description` should probably get an extra `#`.